tests/integration/targets/mso_schema_template_bd/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
       validate_certs: '{{ mso_validate_certs | default(false) }}'
       use_ssl: '{{ mso_use_ssl | default(true) }}'
       use_proxy: '{{ mso_use_proxy | default(true) }}'
-      output_level: '{{ mso_output_level | default("info") }}'
+      output_level: '{{ mso_output_level | default("debug") }}'
```

Comment:
Is the output_level set to `debug` for a reason?

Reply:
Yeah, I was printing debug statements in the code.